openadapt/strategies/mixins/svg_mixin.py

```diff
@@ -0,0 +1,43 @@
+"""
+Implements a ReplayStrategy mixin using Pypotrace to convert an image to SVG text.
+
+NOTE: additional libraries on the system are needed,
+        see https://github.com/flupke/pypotrace#installation
+
+Usage:
+
+    class MyReplayStrategy(SVGReplayStrategyMixin):
+        ...
+"""
+import subprocess
+import numpy as np
+import drawsvg as draw
+
+from openadapt.models import Recording, Screenshot
+from openadapt.strategies.base import BaseReplayStrategy
+
+
+class SVGReplayStrategyMixin(BaseReplayStrategy):
+    """ReplayStrategy mixin that uses Pypotrace to convert an image to SVG text.
```

Comment:
Documentation is stale?

Reply:
Addressed in [7d563f9](https://github.com/MLDSAI/OpenAdapt/pull/198/commits/7d563f99148ff1ab65d6b6b58c78c386fb4fab12)